### ISSUE TITLE
 [The Big Bone Worm]

### ISSUE THREAD

**Issue by Hmudvayne** [dalaranwow/dalaran-wow#4256]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Hmudvayne** [comment]
in Trokkar forest The quest:[The Big Bone Worm] has a bug 
The QUest Item [Fumper] Does not work 
its a Chain quest so you might have to do the first parts 
Regards 
Hmu

**Comment by Rushor** [comment]
DB/Quests: The Big Bone Worm.

Using the drum on a Dead Clefthoof summons some creatures. Eventually it will summon Hai'shulud.

commit: ceb875259a0f8ab5ae044a2648b8b1a4728ae62f

should already be on ptr, dunno maybe it was not applied correctly.

**Comment by Lothloryen** [comment]
@TeridanDW This should be working on PTR, could you check if it got broken, please? Really appreciated.

**Comment by TeridanDW** [comment]
This quest works well on PTR, just a small correction if possible. The summoned creatures from the quest item should have spawn location the spot of the dead Clefthoof, however they spawn directly under the player. That's all.

**Comment by Rushor** [comment]
Yep that fix is already on PTR.
